tools/.shared-functions.sh

```diff
@@ -44,3 +43,20 @@ get_css_file()
     debug "  css file saved ${css_file}"
     echo "${css_file}"
 }
+
+
+check_xcode() {
+  if [[ $(uname) == "Darwin" ]]; then
+    xcode_version=$(xcodebuild -version | grep "Xcode" | awk '{print $2}' | sed 's/[.].*//')
+    minimum_xcode_version=12
+    if [[ $(printf '%s\n' "$xcode_version" "$minimum_xcode_version" | sort -V | head -n1) == "$xcode_version" ]]; then
```

Comment:
Nit: If the `if` clause is empty and the `else` code has the logic, it means the condition the `if` is testing for needs to be reversed